utils/details.js

```diff
@@ -364,10 +366,13 @@ ${
       })
     } catch (error) {
       // add a regular comment if we can't add a review comment
-      await octokit.rest.issues.createComment({
+      //docs.github.com/rest/pulls/comments#create-a-review-comment-for-a-pull-request
+      await octokit.request('POST /repos/{owner}/{repo}/pulls/{pull_number}/comments', {
         ...this.context.repo,
-        issue_number: pull_number,
+        pull_number,
         body,
+        commit_id,
+        path,
```

Comment:
I'm missing something because this is referenced in other parts of the file too, but where is `path` defined? I don't actually see initialization anywhere for a variable named `path`

Reply:
Also `commit_id` , are these auto injected by Actions?

Reply:
`commit_id` is from the payload, yes.
See line 346.

Reply:
:facepalm: I even looked at that and just totally glossed over what that was doing...thanks!